packages/algorand-js/src/errors/errors-list.ts

```diff
@@ -113,6 +113,18 @@ by an index that does not exist.`
     message: "Input is not uint8",
     title: "Input is outside the valid uint8 range of 0 to 255",
     description: `Input is outside the valid uint8 range of 0 to 255`
+  },
+  ASSERT_FIELD_LENGTH: {
+    number: 16,
+    message: "Invalid Field Length - Parse Error, Line : %line% ",
+    title: "Invalid Field Length",
+    description: ``
+  },
+  INVALID_ADDR: {
+    number: 17,
+    message: "Input Parse Error - Invalid Address, Line: %line%",
+    title: "Invalid Address",
+    description: ``
```

Comment:
In the description we should show the invalid address.

Reply:
ok